### PR TITLE
docs: improve documentation

### PR DESCRIPTION
**Pull request by csm-thu**
* creation of a **doc** folder in root folder to regroup all user-oriented README files
* adapt doc to recent changes (config folder refacto)
* use relative links in markdown to point to other files in the same repo
* minor other changes

### CODE REVIEW

File: doc/appConfiguration.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# App behavior configuration
+
+## Edit configuration files
+The file [src/config/AppConfiguration.js](../src/config/AppConfiguration.js) defines constants specific to your application
+instance: you will need to change these values when creating a new staging or prod version of your application.
+
+* **LANGUAGES** is a dict describing the languages available in your webapp (e.g. {en: 'English', fr: 'Français'})
+* **FALLBACK_LANGUAGE** is the default language to use when the user language can't be detected
+* **SCENARIO_STATUS_POLLING_DELAY** is the delay to wait between two requests to the Cosmo Tech API when a scenario is
+  running and the webapp is polling for a status update. Its valie is expressed in milliseconds
```
Comment (COSMO-VMI) [inline doc/appConfiguration.md:10]: Typo: valie -> value

File: doc/appInstance.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# App instance configuration
+
+## Edit configuration files
+The file [src/config/AppInstance.js](../src/config/AppInstance.js) defines constants specific to your application
+instance: you will need to change these values when creating a new staging or prod version of your application.
+
+* **AZURE_TENANT_ID** is the identifier of the Azure tenant your application is registered in.\
+  You can find it in the Azure portal, in the *App registration* blade, in the *Overview* menu.
```
Comment (COSMO-VMI) [inline doc/appInstance.md:8]: I would rather specify menus in the order of operation: Overview menu => App registration

File: doc/appInstance.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+* **APP_REGISTRATION_CLIENT_ID** is the identifier of your app registration.\
+  You can find it in the Azure portal, in the *App registration* blade, in the *Overview* menu.
```
Comment (COSMO-VMI) [inline doc/appInstance.md:10]: Same

File: doc/appInstance.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+* **APPLICATION_INSIGHTS_INSTRUMENTATION_KEY** is the instrumentation key generated by *Azure Application Insights* for
+  your application.\
+  You can find the instrumentation key in the Azure portal, in the *Application Insights* blade, in the *Overview* menu.
```
Comment (COSMO-VMI) [inline doc/appInstance.md:13]: Same

File: doc/appInstance.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  To enable Application Insights in your static web app, please read section "Application Insights" below.
+* **DEFAULT_BASE_PATH** is the URL to the Cosmo Tech API you want to use (e.g. https://dev.api.cosmotech.com) \
```
Comment (COSMO-VMI) [inline doc/appInstance.md:15]: Missing a final dot.

File: doc/appInstance.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  If you don't know which one you should use, please contact your Cosmo Tech administrator
+* **ORGANIZATION_ID** is the identifier of your organization in the Cosmo Tech portal\
```
Comment (COSMO-VMI) [inline doc/appInstance.md:17]: Missing a final dot.

File: doc/appInstance.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+  If you don't know its value, please contact your Cosmo Tech administrator
+* **WORKSPACE_ID** is the identifier of the workspace you created with the Cosmo Tech API\
```
Comment (COSMO-VMI) [inline doc/appInstance.md:19]: Missing a final dot.

File: doc/dashboards.md [hunk 8]
```diff
@@ -1,29 +1,47 @@
...
 
-The Scenario view allows to embed multiple [PowerBI](https://powerbi.microsoft.com/fr-fr/getting-started-with-power-bi/) reports in the app.
+In the web application, the *Scenario* and *Dashboards* pages allow you to embed [PowerBI](https://powerbi.microsoft.com/fr-fr/getting-started-with-power-bi/) reports.
 
-## Scenario entry structure
-To modify entry in the Scenario view menu:
-- Open *webapp_folder*/src/config/Dashboards.js
-- Modify SCENARIO_DASHBOARD_CONFIG array, respecting the following pattern:
+The Scenario view will allow a single report, that is supposed to display the results of the currently selected
+scenario, after it has run. The Dashboards view offer a menu to let users switch between several reports, that can
```
Comment (COSMO-VMI) [inline doc/dashboards.md:6]: Typo: offer -> offers

**Review by COSMO-VMI — CHANGES_REQUESTED** [review]
Some typos and recommendations, other than that, LGTM, GG.

File: doc/appConfiguration.md (hunk 1 above)
Comment (csm-thu) [reply doc/appConfiguration.md]: Fixed, thanks :+1:

File: doc/dashboards.md (hunk 8 above)
Comment (csm-thu) [reply doc/dashboards.md]: Fixed, thanks :+1:

File: doc/appInstance.md (hunk 6 above)
Comment (csm-thu) [reply doc/appInstance.md]: Fixed, thank you :+1:

File: doc/appInstance.md (hunk 5 above)
Comment (csm-thu) [reply doc/appInstance.md]: Fixed, thank you :+1:

File: doc/appInstance.md (hunk 7 above)
Comment (csm-thu) [reply doc/appInstance.md]: Fixed, thank you :+1:

File: doc/appInstance.md (hunk 2 above)
Comment (csm-thu) [reply doc/appInstance.md]: Replaced by `Azure portal > App registration > Overview` for clarity. Thank you :+1:

File: doc/appInstance.md (hunk 3 above)
Comment (csm-thu) [reply doc/appInstance.md]: Replaced by `Azure portal > App registration > Overview` for clarity. Thank you :+1:

File: doc/appInstance.md (hunk 4 above)
Comment (csm-thu) [reply doc/appInstance.md]: Replaced by `Azure portal > Application Insights > Overview` for clarity. Thank you :+1:

**Pull request merged**